### PR TITLE
doc: adding meeting notes Sep 24 and Oct 1

### PR DESCRIPTION
**Pull request by joesepi**
Refs: #336
Refs: #329

### CODE REVIEW

**Comment by joesepi** [comment]
We can *fast track* this if we get two approvals and two thumbs on this comment. Not necessary, but I'm happy to move it along to keep down on noise. Thanks.

File: meetings/2019-09-24.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  * Application is private by default as submitted, but after approval application can be made public with consent of both parties
+ * It would possibly be helpful to have a short list of canonical approved applications to provide examples & ways to encourage other applicants
+ * Action Item: propose some text to clarify that the application is private; possibly iterate on language that encourages a process to make public versions of the applications available.
+
+* Administering Project Applications [#331](https://github.com/openjs-foundation/cross-project-council/issues/331)
+  * Would be great if there were tooling we can take advantage of, in order to provide some helpful ‘pokes’ and some metrics around how we’re doing.
+  * Private GH project repo & other ideas discussed. Folks will continue to explore & discuss
+
+* Post-Bootstrap work to be managed by CPC [#115](https://github.com/openjs-foundation/cross-project-council/issues/115)
+  *
```
Comment (mhdawson) [inline meetings/2019-09-24.md]: ```suggestion```
  * We walked through the board to review status of each item"

File: meetings/2019-10-01.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    * Will wait for other projects to work through process before doing this
+
+
+* Have opened up proposal for call for submissions to collaborator summit
+  * https://github.com/openjs-foundation/summit/issues/new/choose

+  * Will send out email on mailing list
+
+
+
```
Comment (mhdawson) [inline meetings/2019-10-01.md]:
```suggestion
```

File: meetings/2019-10-01.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  * Provide guidance on how to refer to the foundation in licenses/copyright statements #258
+    * Brian: process being made; blocked on legal documentation mentioned earlier
+  * Create Mock-ups for displaying OpenJS Foundation mark on project websites #266
+    * Not going to give guidance; too much to worry about each individual project
+  * Forum for keeping certain project details #265
+  * Ask projects to start updating references to docs in cpc repo (ex coc etc.) #282
+    * Will wait for other projects to work through process before doing this
+
+
+* Have opened up proposal for call for submissions to collaborator summit
```
Comment (mhdawson) [inline meetings/2019-10-01.md]:
```suggestion
```

File: meetings/2019-10-01.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+[4](https://github.com/openjs-foundation/cross-project-council/projects/4)
+  * Provide guidance on how to refer to the foundation in licenses/copyright statements #258
+    * Brian: process being made; blocked on legal documentation mentioned earlier
+  * Create Mock-ups for displaying OpenJS Foundation mark on project websites #266
+    * Not going to give guidance; too much to worry about each individual project
+  * Forum for keeping certain project details #265
+  * Ask projects to start updating references to docs in cpc repo (ex coc etc.) #282
+    * Will wait for other projects to work through process before doing this
+
+
```
Comment (mhdawson) [inline meetings/2019-10-01.md]:
```suggestion
```

File: meetings/2019-10-01.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+* Have opened up proposal for call for submissions to collaborator summit
+  * https://github.com/openjs-foundation/summit/issues/new/choose

+  * Will send out email on mailing list
+
+
```
Comment (mhdawson) [inline meetings/2019-10-01.md]:
```suggestion
```

File: meetings/2019-10-01.md (hunk 2 above)
Comment (mhdawson) [inline meetings/2019-10-01.md]:
```suggestion
```

**Pull request merged**